src/components/og-spinner/og-spinner.scss

```diff
@@ -7,6 +7,17 @@
 // -----------------------------------------------------------------------------
 
 :host {
+    /**
+     * @prop --Width:
+     * Primary Color of the spinner.
+     */
+    --Width: 32px;
+    /**
+     * @prop --Height:
+     * Primary Color of the spinner.
+     */
+    --Height: 32px;
```

Comment:
Comments need to be updated.
Why is it just `--Width` and not `--og-spinner-Width` (same for `--Height`)?